### DISCUSSION TITLE
Python Version

### DISCUSSION THREAD

**Discussion by aklund21** [medspacy/medspacy#221] [Q&A]
Is there any version of Medspacy that works with python < 3.8? If so, any downsides such as missing functionality? I have some system limitations. Have tried installing back to 1.0.0 from current without success on new python 3.7 (linux). Thanks.

**Comment by burgersmoke** [comment]
Good question @aklund21 .  Currently, medspacy requires Python 3.8 or greater, since some changes (like f-strings) required this.

**Reply by aklund21** [reply]
Thought there was some other dependency but doesn't hurt to ask. Appreciate the quick feedback. Tried to install back to 0.1.0.0. Pypi usually just specifies the latest release requirements in Release History.

**Reply by burgersmoke** [reply]
Absolutely.  It's a great question so it's worth asking.  I think on releases before 1.0.0.0, we didn't specify which version(s) were or were not supported.  We'll try to do better with that going forward.
